deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -12,6 +12,11 @@
   notify:
     - Restart Nginx
 
+- name: Download Jenkins CLI jar
+  get_url:
```

Comment:
This already exists at `/var/cache/jenkins/war/WEB-INF/jenkins-cli.jar` on the VM. Can we just use that path?